easybuild/easyconfigs/g/gc/gc-8.2.2-GCCcore-12.3.0.eb

```diff
@@ -0,0 +1,42 @@
+easyblock = 'ConfigureMake'
+
+name = 'gc'
+version = '8.2.2'
```

Comment:
Is there a reason to not use [8.2.4](https://github.com/ivmai/bdwgc/releases/tag/v8.2.4)?

Reply:
my bad, I was looking at https://hboehm.info/gc/gc_source which has 8.2.2 as the latest one.

Reply:
updated versions of `gc` and `libatomic`